src/ripple/app/paths/cursor/DeliverNodeReverse.cpp

```diff
@@ -238,7 +244,8 @@ TER PathCursor::deliverNodeReverse (
             resultCode = increment(-1).deliverNodeReverse(
                 node().offerOwnerAccount_,
                 saInPassReq,
-                saInPassAct);
+                saInPassAct,
+                /*restart dir*/false);
```

Comment:
Wait - is this right?  Did I miss this explanation before as to why `restartDir` is always `false` when we call the function recursively?

If this is right, probably there should be an explanation here anyway.